src/gui/editors/PianoRoll.cpp

```diff
@@ -3522,6 +3522,23 @@ void PianoRoll::finishRecordNote(const Note & n )
 						it->key(), it->getVolume(),
 						it->getPanning() );
 				n1.quantizeLength( quantization() );
+
+				//Get selected chordModel
+				const InstrumentFunctionNoteStacking::Chord & chord = InstrumentFunctionNoteStacking::ChordTable::getInstance()
+						.getChordByName( m_chordModel.currentText() );
+
+					if( ! chord.isEmpty() )
```

Comment:
It'd be better to unindent the `if` block one level.

Reply:
Also, `if( a )` is an old style. According to [our coding convention](https://github.com/LMMS/lmms/wiki/Coding-conventions), `if (a)` is correct. However, you can leave this as-is if the change breaks consistency.